SpatialGDK/Source/SpatialGDKTests/Private/RPCContainer/RPCContainerTest.cpp

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestDefinitions.h"
+
+#include "ObjectDummy.h"
+#include "ObjectSpy.h"
+
+#include "Utils/RPCContainer.h"
+#include "Schema/RPCPayload.h"
+
+#include <Core.h>
+
+#define RPCCONTAINER_TEST(TestName) \
```

Comment:
Why is this needed?

Reply:
I think it would be easier to be able to see the name directly.

Reply:
It should be more flexible with macro, if we decide to modify the test configuration. So I'd keep it.

Reply:
I'm not sure what flexibility it adds. But it doesn't really matter,

Reply:
Should I resolve it then?